### PR TITLE
fix(curriculum): clarify lesson explanation

### PR DESCRIPTION
**Pull request by shawn-carrie**
The instructions on this lesson were confusing and not very beginner-friendly, so I've added some clarifying language.

From the JavaScript Basics curriculum:
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups

### CODE REVIEW

**Review by Sboonny — CHANGES_REQUESTED** [review]
when I checked the content on [grammerly report](https://www.grammarly.com/report?alerts=NobwRAxghgLgpgcwPYCcCeYBcYDCSC2ADgAoCuAdhDGADRgIpKmFZhmUymwCWS5tkWIlRoAEqXxR%2B2dlS4xe5AATdlEAoSbkAJgHp1RADZwAHkoDOccvEpxzA9RWqYAjAF8a4aPGTpWAETh4FHxVOBR7OgYmFmwAcRQofEkUByFfMQkpAKDw0PJwpVJLJQAKKF0pXRgACzhqmu5zGiUgiAA6AEoHLWd3T0EfEVZiQygEbigUJvwBaOYRsYmpmbSh9HFJaTAAVXJUbgnyKEMldWsrajpHaywXAE4PL3Th7AB1VG0cGqRuCDg5owFtgAKLkGpSf74S5rYQbLLbD4obRnH5-AHXXpYABMbgAukA&page=plagiarism&breadcrumbs=true) it showed plagriarism. 

Is this text copied from somewhere?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md [hunk 1]
```diff
@@ -25,18 +29,22 @@ const alpha = {
...
 ```
 
 `thirdLetter` is the string `Y`, `lastLetter` is the string `C`, and `valueLookup` is the string `Y`.
 
 # --instructions--
 
-Convert the switch statement into an object called `lookup`. Use it to look up `val` and assign the associated string to the `result` variable.
+Instead of using a bulky `switch` statement, let's swap this code for a more lightweight object to perform our lookup operation.
+
+Convert the switch statement into an object called `lookup` – which will store coded words from the [The International Radiotelephony Spelling Alphabet](https://en.wikipedia.org/wiki/NATO_phonetic_alphabet).
```
Comment (Sboonny) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:45]: We are against the usage of external links, we are trying to make freeCodeCamp self sufficient

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md [hunk 2]
```diff
@@ -25,18 +29,22 @@ const alpha = {
   26:"A"
 };
 
-const thirdLetter = alpha[2];
-const lastLetter = alpha[24];
+const thirdLetter = reverseAlphabet[2];
+const lastLetter = reverseAlphabet[25];
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:33]: The change from `alpha` to `reverseAlphabet` is the only part of this PR that I think we should keep, honestly

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md [hunk 3]
```diff
@@ -25,18 +29,22 @@ const alpha = {
...
 
-const value = 2;
-const valueLookup = alpha[value];
+let value = 2;
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md:35]: No, `const` is correct here

**Comment by Sboonny** [comment]
Thank you for your pull request, you can open new pull requests with jeremy suggestion if you want.

But we are closing this pull request. Please suggest links and other details to add the challenge's corresponding guide post through [a forum topic](https://forum.freecodecamp.org/new-topic?category=Contributors&title=&body=**What%20is%20your%20hint%20or%20solution%20suggestion%3F**%0A%0A%0A%0A%0A**Challenge%3A**%0A%0A%0A**Link%20to%20the%20challenge%3A**) instead.

If you think we're wrong in closing this pull request, please request for it to be reopened. Thank you, and happy coding.

**Pull request closed without merging**